Makefile

```diff
@@ -18,6 +18,10 @@ DIST := dist
 DIST_DIRS := $(DIST)/binaries $(DIST)/release
 IMPORT := code.gitea.io/gitea
 
+# By default use go's 1.25 experimental json v2 library when building
+# TODO: remove when no longer experimental
+export GOEXPERIMENT ?= jsonv2
```

Comment:
fyi we do not use submake, so this `export` is useless.

Reply:
Yes, all commands will see the variables, `export` in a makefile is only for sub-make:

https://www.gnu.org/software/make/manual/html_node/Variables_002fRecursion.html#Communicating-Variables-to-a-Sub_002dmake

Reply:
Are you sure or have you tested? I don't think the the GNU toolchain can have such a counterintuitive, global-polluting and dirty design.

<img width="424" height="598" alt="image" src="https://github.com/user-attachments/assets/78dc916a-b795-4583-8c13-68c2b7d9fabb" />

Reply:
I agree that "export" is not the best approach, while it just works, so I just kept the original change from techknowlogick's https://github.com/go-gitea/gitea/pull/35392/commits/94d537c8450392bc6a77ce0c0d2627f6908d69ae